library/cwm/src/lib/cwm/widget.rb

```diff
@@ -98,6 +98,8 @@ def self.widget_type=(type)
     #   @return [String] translated label text for the widget
 
     # @!method opt
+    #   Specifies options passed to widget. When {handle} method defined, then
```

Comment:
well, it not need to be always defined and e.g. custom widget do it different way. So that is reason why I use often. so what about
```
When {handle} method is defined, it often expected to be immediatelly notified from widget and for that purpose opt with `[:notify]` as return value is needed.
```